net-p2p/ppcoind/ppcoind-0.4.0-r1.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DB_VER="4.8"
+
+inherit db-use eutils systemd user
+
+MyPV="${PV/_/-}"
+MyPN="ppcoin"
+MyP="${MyPN}-${MyPV}"
+
+DESCRIPTION="Cryptocurrency forked from Bitcoin which aims to be energy efficiency"
+HOMEPAGE="http://peercoin.net/"
+SRC_URI="mirror://sourceforge/${MyPN}/${MyP}-linux.tar.gz -> ${MyP}.tar.gz"
+
+LICENSE="MIT ISC GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples ipv6 logrotate upnp"
+
+RDEPEND="
+	dev-libs/boost[threads(+)]
+	dev-libs/openssl:0[-bindist]
+	logrotate? (
+		app-admin/logrotate
+	)
+	upnp? (
+		net-libs/miniupnpc
+	)
+	sys-libs/db:$(db_ver_to_slot "${DB_VER}")[cxx]
+"
+DEPEND="${RDEPEND}
+	>=app-shells/bash-4.1
+	sys-apps/sed
+"
+
+DOCS=( ../README README.md )
+
+PATCHES=( "${FILESDIR}"/${P}-gcc6.patch )
+
+S="${WORKDIR}/${MyP}-linux/src"
+
+pkg_setup() {
+	local UG='ppcoin'
+	enewgroup "${UG}"
+	enewuser "${UG}" -1 -1 /var/lib/ppcoin "${UG}"
+}
+
+src_prepare() {
+	default
+
+	if has_version '>=dev-libs/boost-1.52'; then
+		sed -i 's/\(-l db_cxx\)/-l boost_chrono$(BOOST_LIB_SUFFIX) \1/' src/makefile.unix
+	fi
```

Comment:
just make this unconditional (or if you'd like to do me a favour, turn it into an unconditional patch)